app/Http/Controllers/CommunityController.php

```diff
@@ -65,8 +65,9 @@ public function getLive(Request $request)
 
         $featuredStreamId = Cache::get('featuredStream');
         if ($featuredStreamId !== null) {
+            $featuredStreamId = floatval($featuredStreamId);
             foreach ($streams as $stream) {
-                if ($stream->_id !== $featuredStreamId) {
+                if (floatval($stream->_id) !== $featuredStreamId) {
```

Comment:
btw, if int overflows, what's in `$stream->_id` to begin with? If it's already int, I don't see how casting will help?

I think it's more like victim of strict comparison upgrade and `Cache` casts everything to string.